harvester/load.py

```diff
@@ -93,14 +105,17 @@ def simple_transform(dcatus_catalog):
         "title": "title",
     }
 
-    for k, v in dcatus_catalog.items():
+    for k, v in dcatus_dataset.items():
         if k not in mapping:
             continue
         if isinstance(mapping[k], dict):
             temp = {}
+            to_skip = ["@type"]
             for k2, v2 in v.items():
                 if k2 == "hasEmail":
                     v2 = get_email_from_str(v2)
+                if k2 in to_skip:
+                    continue
                 temp[mapping[k][k2]] = v2
             output = {**output, **temp}
```

Comment:
is there any chance we overwrite good values in `**output` with values in `**temp`?

Reply:
at the moment, as long as `fn` and `hasEmail` don't exist in the root of the dataset we should be fine but this is something good to look out for when we continue to refine this method and add more data to process in `mapping` and `output`